source/examples/types/MceStartStopIO/definition.yaml

```diff
@@ -73,48 +101,40 @@ var:
     default_value: "0"
     comment: "input: stop and flush the motion queue (uses rising edge)"
 
-  - name: bSystemReady
+  - name: bServoOn
     description: |
       *Used as output*
 
-      This tells the system is ready for processing motion commands.
+      This tells the servos are enabled.
+      The system is ready for processing motion commands.
 
       This output can be used as condition by lower level state machines which
       issue motion commands (e.g. PosTable).
 
     type: BOOL
-    comment: "output: system ready for processing motion commands"
+    comment: "output: servos are enabled and system is ready for processing motion commands"
 
-  - name: bStartIndicator
+  - name: bIdle
     description: |
       *Used as output*
 
-      Connect this signal to the indicator light of the *start button*.
+      The robot controller servos are off.
```

Comment:
Perhaps for consistency you could use the same text as elsewhere for `bIdle`, with "(robot controller servos off") added?